docs/client-sdk/client-sdk-cse.md

```diff
@@ -0,0 +1,113 @@
+# FIDO Device Onboard (FDO) Client SDK CSE API Reference
+
+This document describes the list of all the functions used by the client SDK to interact with the CSE (Converged Security Engine).
```

Comment:
use Intel<sup>&reg;</sup> CSE instead for using just CSE

Reply:
Done